helm/reana/Chart.yaml

```diff
@@ -26,7 +26,7 @@ keywords:
   - reusable-science
 type: application
 # Chart version.
-version: 0.8.0-alpha.1
+version: 0.8.0-alpha.2
```

Comment:
FYI testing locally using released Docker Hub images (i.e. repulled everything!) and:

```console
$ helm install  reana helm/reana --wait
```

The only remaining thing is to release Helm chart.